CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
```

Comment:
Buraya türkçe olarak ne karşılık bulabiliriz acaba? Kendim düşünmek için not alıyorum